transit/rest_api/forms/pod_variance.py

```diff
@@ -54,6 +57,20 @@ def create(self, validated_data):
             PODVarianceDetails(**details_dict).save()
         return pod_variance  # noqa: R504
 
+    @transaction.atomic
+    def update(self, instance, validated_data):
+        # firstly update pod variance details objects
```

Comment:
I need such kind of extension for PODVariance  'update' to allow update PodVarianceDetails. This solution allows me to use single call of PUT/PATCH update for updating PodVariance with PodVarianceDetails on frontend side. 

Moreover I decided to update PodVarianceDetails objects without deleting and creating again such objects because in PodVarianceDetails we can only update 'quantity'. There is no possibility to add/remove items from this list.

Reply:
Thanks for explanation